src/aioswitcher/device/__init__.py

```diff
@@ -359,6 +374,17 @@ class SwitcherDualShutterSingleLightBase(ABC):
     lights: DeviceState
```

Comment:
Same here, missed in previous review, I think it might be better to call it `light`